### DISCUSSION TITLE
How can I set a Displayname only for one player?

### DISCUSSION THREAD

**Discussion by FloGamerMC** [Minestom/Minestom#1033] [Help] [answered]
I want to set a display name of a player for each other player individually. How can I do this?

**Comment by GreatWyrm** [comment, marked as answer]
Use [Player#setDisplayName()](https://javadoc.minestom.net/net/minestom/server/entity/Player.html#setDisplayName(net.kyori.adventure.text.Component)).

**Reply by FloGamerMC** [reply]
I want to set it individually for each Player. For example player1 see the name of a player als Steve and player2 see the name of the same player as Alex.

**Reply by GreatWyrm** [reply]
Ah, in that cause you would want to send a Packet to player1 and a different packet to player2 with this:

`new PlayerInfoPacket(PlayerInfoPacket.Action.UPDATE_DISPLAY_NAME, new PlayerInfoPacket.UpdateDisplayName([uuid of target], displayName));`

So send 2 separate packets and change the displayName variable for each one
